apps/core/src/migration/migration.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { Controller, Post } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { MigrationService } from './migration.service';
+
+@ApiTags('migration')
+@Controller('api/migration')
+export class MigrationController {
+  constructor(private readonly migrationService: MigrationService) {}
+
+  @Post()
+  public migration() {
+    // Purposely don't use await so this request can process after the response
+    return this.migrationService.migrate();
```

Comment:
nit: `migrate()` returns nothing, better not `return` the value of it here, unless there's a return value; having it here implies the response rely on it